submissions/mksen85/style.css

```diff
@@ -0,0 +1,196 @@
+* {
+    padding: 0;
+    margin: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+
+.wrapper{
+    height: 300px;
+    padding: 20px;
+}
+
+.header {
+  position: fixed;
+  max-height: 55px;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  top: 0;
+  left: 0;
+  background-color: rgb(190, 232, 232);
+}
+
+.header_logo {
+  max-width: 100%;
+  border-radius: 50%;
+  overflow: hidden;
+  padding-left: 10px;
+}
+
+
+.menu_list{
+  display: flex;
+  padding: 10px;
+  font-size: 20px;
+  }
+  
+  .menu a{
+    text-decoration: none;
+    color: rgb(99, 93, 93);
+    font-weight: bold;
+    align-items: center;
+  }
+
+ .menu_link{
+    padding-left: 15px;
+    padding-right: 10px;
+    border: 1em;
+    height: 24p;
+    position: relative;
+    display: grid;
+    width: 100%;
+    z-index: 30;
+    align-items: center;
+    border: 1em;
+  }
+
+  .menu_list li{
+    list-style: none;
+  }
+
+  menu li{
+    position: relative;
+  }
```

Comment:
`menu` tag?)
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)